### PR TITLE
ReadMe-GRD74196

### PR DESCRIPTION
**Pull request by apurva-birajdar**
Following plugin's ReadMe files are updated to be consistent with other plugins.
1.MongoDB
2.MySQL
3.MySQL Percona
4.S3
5.HDFS

Below changes :
Removed instructions of how to build gem files and configure Logstash locally.
Added missing steps to be consistent with other ReadMe files.

### CODE REVIEW

File: filter-plugin/logstash-filter-hdfs-guardium/README.md [hunk 1]
```diff
@@ -1,161 +1,120 @@
...
-2020-11-29 10:44:07,519 INFO FSNamesystem.audit: allowed=true	ugi=hbase (auth:SIMPLE)	ip=/10.11.18.117	cmd=create	src=/hbase/MasterProcWALs/pv2-00000000000000000004.log	dst=null	perm=hbase:hbase:rw-r--r--	proto=rpc	callerContext=CLI
-2020-11-29 10:44:07,523 INFO FSNamesystem.audit: allowed=true	ugi=hbase (auth:SIMPLE)	ip=/10.11.18.117	cmd=listStatus	src=/hbase/MasterProcWALs	dst=null	perm=null	proto=rpc	callerContext=CLI
-2020-11-29 10:44:07,544 INFO FSNamesystem.audit: allowed=true	ugi=hbase (auth:SIMPLE)	ip=/10.11.18.117	cmd=create	src=/hbase/MasterProcWALs/pv2-00000000000000000005.log	dst=null	perm=hbase:hbase:rw-r--r--	proto=rpc	callerContext=CLI
-2020-11-29 10:44:07,550 INFO FSNamesystem.audit: allowed=true	ugi=hbase (auth:SIMPLE)	ip=/10.11.18.117	cmd=listStatus	src=/hbase/MasterProcWALs	dst=null	perm=null	proto=rpc	callerContext=CLI
-```
 
-## Limitations
+### Before you begin
+• You must have LFD policy enabled on the collector. The detailed steps can be found in step 4 [on this page](https://www.ibm.com/docs/en/guardium/11.4).
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-hdfs-guardium/README.md:90]: Please ONLY work with plug-in read-mes that are currently live in GitHub. That way we keep all the changes and corrections that have been made

Comment (AlexaNahum) [inline filter-plugin/logstash-filter-hdfs-guardium/README.md:90]:
```suggestion
•Configure the policies you require. See [policies](/docs/#policies) for more information.
```

File: filter-plugin/logstash-filter-mysql-percona-guardium/README.md [hunk 2]
```diff
@@ -4,81 +4,121 @@
...
+
+## 1. Configuring the Mysql server
+
+It is an extension of MySql-Guardium Logstash filter plug-in. see [MySql-Guardium Logstash filter plug-in](https://github.ibm.com/Activity-Insights/universal-connectors/blob/master/filter-plugin/logstash-filter-mysql-guardium/README.md)
+
+## 2. Installing and enabling auditing
+
+### Percona configuration
+
+1. On the database, update the file: /etc/percona-server.conf.d/mysqld.cnf
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mysql-percona-guardium/README.md:20]:
```suggestion
1. On the database, update the file: /etc/percona-server.conf.d/mysqld.cnf.
```

File: filter-plugin/logstash-filter-mysql-percona-guardium/README.md [hunk 3]
```diff
@@ -4,81 +4,121 @@
...
+   https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-installation-configuration.html#installation
+
+## b. Filebeat configuration:
+
+To use Logstash to perform additional processing on the data collected by Filebeat, we need to configure Filebeat to use Logstash. To do this, modify the filebeat.yml file which you can find inside the folder where Filebeat is installed. Follow these instructions for finding the installation directory:
+https://www.elastic.co/guide/en/beats/filebeat/current/directory-layout.html
+
+### Procedure:
+
+1. Configuring the input section :-
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mysql-percona-guardium/README.md:49]:
```suggestion
1. Configuring the input section :
```

File: filter-plugin/logstash-filter-mysql-percona-guardium/README.md [hunk 4]
```diff
@@ -4,81 +4,121 @@
...
+
+   • Locate "filebeat.inputs" in the filebeat.yml file, then add the following parameters.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mysql-percona-guardium/README.md:51]:
```suggestion
   • Locate "filebeat.inputs" in the filebeat.yml file, and then add the following parameters.
```

File: filter-plugin/logstash-filter-mysql-percona-guardium/README.md [hunk 5]
```diff
@@ -4,81 +4,121 @@
...
-    #&zwnj;- c:\programdata\elasticsearch\logs*
-    C:\downloads\docker_volumes*
+    - c:\programdata\elasticsearch\logs*
+    - C:\downloads\docker_volumes*
+  
+    ```
+
+2. Configuring the output section:
+
+   • Locate "output" in the filebeat.yml file, then add the following parameters.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mysql-percona-guardium/README.md:65]:
```suggestion
   • Locate "output" in the filebeat.yml file, and then add the following parameters.
```

File: filter-plugin/logstash-filter-mysql-percona-guardium/README.md [hunk 6]
```diff
@@ -4,81 +4,121 @@
...
+    ```
+   In addition, events are configured with the add_locale, add_host_metadata, and add_tags processors (to add an "hdfs" tag).
 
-</ol>
-</li>
 
-<li class="bx--list__item">Restart the filebeat service by
-entering:<div class="bx--snippet bx--snippet--multi bx--snippet--wraptext bx--snippet--expand"><div class="bx--snippet-container" tabindex="0" aria-label="Code snippet"><pre class="codeblock"><code class="language-plaintext hljs">sudo service filebeat restart</code></pre></div><button data-copy-btn="" class="bx--copy-btn" aria-label="Copy to clipboard" title="Copy to clipboard" type="button" tabindex="0"><svg focusable="false" preserveAspectRatio="xMidYMid meet" style="will-change: transform;" xmlns="http://www.w3.org/2000/svg" class="bx--snippet__icon" width="16" height="16" viewBox="0 0 16 16" aria-hidden="true"><path d="M14,5v9H5V5h9m0-1H5A1,1,0,0,0,4,5v9a1,1,0,0,0,1,1h9a1,1,0,0,0,1-1V5a1,1,0,0,0-1-1Z"></path><path d="M2,9H1V2A1,1,0,0,1,2,1H9V2H2Z"></path></svg>
-      </button></div></li>
+3. To learn more about Filebeat processors [here](https://www.elastic.co/guide/en/beats/filebeat/current/filtering-and-enhancing-data.html#using-processors).
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mysql-percona-guardium/README.md:86]:
```suggestion
3. To learn more about Filebeat processors, click [here](https://www.elastic.co/guide/en/beats/filebeat/current/filtering-and-enhancing-data.html#using-processors).
```

File: filter-plugin/logstash-filter-mysql-percona-guardium/README.md [hunk 7]
```diff
@@ -4,81 +4,121 @@
...
-      </button></div></li>
+The Guardium universal connector is the Guardium entry point for native audit logs.
+The universal connector identifies and parses received events, and converts them to a standard Guardium format.
+The output of the Guardium universal connector is forwarded to the Guardium sniffer on the collector, for policy and auditing enforcements.
 
-<li class="bx--list__item">Restart the mysql service.</li>
+### Before you begin
 
-</ol>
+• You must have LFD policy enabled on the collector. The detailed steps can be found in step 4 [on this page](https://www.ibm.com/docs/en/guardium/11.4).
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mysql-percona-guardium/README.md:97]:
```suggestion
• Configure the policies you require. See [policies](/docs/#policies) for more information.
```

File: filter-plugin/logstash-filter-mysql-percona-guardium/README.md [hunk 8]
```diff
@@ -4,81 +4,121 @@
...
 
-</section>
-</div>
+• You must have permission for the S-Tap Management role. The admin user includes this role by default.
 
-## Configuring the MySQL filters in Guardium Insights
-To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
-In the input configuration section, refer to the Filebeat section.
+• Download the [mysql-percona-offline-plugin.zip](./MysqlPerconaOverFilebeatPackage/mysql-percona-offline-plugin.zip).
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mysql-percona-guardium/README.md:101]:
```suggestion
• Download the [mysql-percona-offline-plugin.zip](./MysqlPerconaOverFilebeatPackage/mysql-percona-offline-plugin.zip). (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.

```

File: filter-plugin/logstash-filter-mysql-percona-guardium/README.md [hunk 9]
```diff
@@ -4,81 +4,121 @@
...
-## Contribute
-You can enhance this filter and open a pull request with suggested changes - or you can use the project to create a different filter plug-in for Guardium that supports other data sources.
+1. On the collector, go to Setup > Tools and Views > Configure Universal Connector.
 
-## References
-See [documentation for Logstash Java plug-ins](https://www.elastic.co/guide/en/logstash/current/contributing-java-plugin.html).
+2. First enable the Universal Guardium connector, if it is disabled already.
 
-See [Guardium Universal connector commons](https://www.github.com/IBM/guardium-universalconnector-commons) library for more details regarding the standard Guardium Record object.
+4. Click **Upload File** and select the offline [mysql-percona-offline-plugin.zip](./MysqlPerconaOverFilebeatPackage/mysql-percona-offline-plugin.zip) file. After it is uploaded, click OK.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mysql-percona-guardium/README.md:109]:
```suggestion
4. Click **Upload File** and select the offline [mysql-percona-offline-plugin.zip](./MysqlPerconaOverFilebeatPackage/mysql-percona-offline-plugin.zip) file. After it is uploaded, click OK. This is not necessary for Guardium Data Protection v12.0 and later.	
```

File: filter-plugin/logstash-filter-mysql-percona-guardium/README.md [hunk 10]
```diff
@@ -4,81 +4,121 @@
...
+
+7. Update the input section to add the details from the [perconaFilebeat.conf](./perconaFilebeat.conf) file's input part, omitting the keyword "input{" at the beginning and its corresponding "}" at the end. Provide required details for DB server name, username and password for making JDBC connectivity.
+
+8. Update the filter section to add the details from the [perconaFilebeat.conf](./perconaFilebeat.conf) file's filter part, omitting the keyword "filter{" at the beginning and its corresponding "}" at the end. Provide the same DB server name as in above step against the Server_Hostname attribute in the filter section.
+
+9. Click Save. Guardium validates the new connector, and enables the universal connector if it was disabled. After it is validated, it appears in the Configure Universal Connector page.
+
+
+## Configuring the MySQL filters in Guardium Insights
+To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mysql-percona-guardium/README.md:123]:
```suggestion
To configure this plug-in for Guardium Insights, follow [this guide](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md).
```

**Review by AlexaNahum — APPROVED** [review]

**Review by ofer-haim — APPROVED** [review]

**Pull request merged**